package/yast2-vm.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Wed Oct 11 14:59:40 UTC 2017 - aginies@suse.com
+
+- use new pattern name for SLES (same as TW)
```

Comment:
Please add the version to the change log.
OBS will probably also want to have a Bugzilla or Fate number.